File: 00. Estructura/01. Lenguajes de proposito general/02. Programación básica/10. Excepciones/01. Creación/JavaScript/Creación.js

// Crear excepción
class MiExcepcion extends Error {
    // Contenido
}

// Crear excepción sobreescribiendo los constructores
class MiExcepcion extends Error {
    
    // Con mensaje y sin error de causa
    constructor(message) {
        super(message);
        // Contenido
        super.name = "nombre";      // Cambiar nombre
        super.message = "mensaje";  // Cambiar mensaje
        super.stack = "traza";      // Cambiar traza
        // Contenido
    }
    
}

File: 00. Estructura/01. Lenguajes de proposito general/02. Programación básica/10. Excepciones/01. Creación/Python/Creación.py

# Crear excepción
class MiExcepcion(Exception):
    # Contenido

# Crear excepción sobreescribiendo los constructores
class MiExcepcion(Exception):
    
    # Con mensaje
    def __init__(self, message):
        super().__init__(message)
        # Contenido